### PR TITLE
[Lang] Fix pylint rule W0611

### PR DESCRIPTION
**Pull request by tczhangzhi**
Related issue = #3151

### CODE REVIEW

**Comment by tczhangzhi** [comment]
Comments on `python/taichi/lang/shell.py:10:4` and `python/taichi/lang/shell.py:17:4` are as follows.

We suppose importing this package is necessary here because we need to check the independence and ensure `sourceinspect` exists. So we use `# pylint: disable=unused-import` to neglect the pylint error of these two lines:
```
try:
    import sourceinspect as oinspect # pylint: disable=unused-import
except ImportError:
    warn('`sourceinspect` not installed!')
    warn(
        'Without this package Taichi may not function well in Python IDLE interactive shell, '
        'Blender scripting module and Python native shell.')
    warn('Please run `python3 -m pip install sourceinspect` to install.')
    import inspect as oinspect # pylint: disable=unused-import
```

**Review by sjwsl — COMMENTED** [review]
Wildcard import is not allowed after #3471

File: python/taichi/ui/ui.py [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-from taichi.type.annotations import ext_arr, template
 
 if _ti_core.GGUI_AVAILABLE:
 
     from .camera import Camera
-    from .canvas import Canvas
-    from .gui import Gui
-    from .scene import Scene
-    from .window import Window
+    from .constants import *
```
[the remark this answers is not in the document]
Comment (tczhangzhi) [reply python/taichi/ui/ui.py]: k, thanks for your guidance

File: python/taichi/ui/gui.py [hunk 2]
```diff
@@ -1,11 +1,6 @@
-import pathlib
 from contextlib import contextmanager
 
-from taichi.core import ti_core as _ti_core
-from taichi.lang.impl import default_cfg
-from taichi.lang.kernel_impl import kernel
-from taichi.lang.ops import get_addr
-from taichi.type.annotations import ext_arr, template
+from .utils import *
```
[the remark this answers is not in the document]
Comment (tczhangzhi) [reply python/taichi/ui/gui.py]: DONE

**Comment by sjwsl** [comment]
Could you also add this pylint rule in CI to verify your work? See [here](https://github.com/taichi-dev/taichi/issues/3151#issuecomment-965944567) for more details. Thanks!

**Comment by tczhangzhi** [comment]
> Could you also add this pylint rule in CI to verify your work? See [here](https://github.com/taichi-dev/taichi/issues/3151#issuecomment-965944567) for more details. Thanks!

Okay, my bad ( ͒˃⌂˂ ͒)

**Comment by tczhangzhi** [comment]
Fix bugs by preserving `Camera`, `Canvas` and other importation:
```
from .camera import Camera # pylint: disable=unused-import
from .canvas import Canvas # pylint: disable=unused-import
from .constants import * # pylint: disable=unused-import
from .gui import Gui # pylint: disable=unused-import
from .scene import Scene # pylint: disable=unused-import
from .window import Window # pylint: disable=unused-import
```
We need to check `_ti_core.GGUI_AVAILABLE` and import `Scene`, `Window`, `constants`, and others for utilization in a fashion of:
```
window = ti.ui.Window('Window Title', (640, 360))
scene = ti.ui.Scene()
ti.ui.PRESS
```

File: python/taichi/ui/ui.py [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
 if _ti_core.GGUI_AVAILABLE:
 
-    from .camera import Camera
-    from .canvas import Canvas
-    from .gui import Gui
-    from .scene import Scene
-    from .window import Window
+    from .camera import Camera # pylint: disable=unused-import
+    from .canvas import Canvas # pylint: disable=unused-import
+    from .constants import * # pylint: disable=unused-import
```
Comment (sjwsl) [inline python/taichi/ui/ui.py:7]:
```suggestion
    from .constants import * # pylint: disable=unused-import, disable=wildcard-import
```

Comment (tczhangzhi) [reply python/taichi/ui/ui.py]: Got it!

**Review by ailzhang — APPROVED** [review]
Thanks for fixing this!

**Comment by lucywsq** [comment]
Hi,@tczhangzhi We are so appreciative of your contribution to fixing _pylint_ rules through the “Let’s become Taichi contributors within 10 minutes” program. Please send an email to  siqiwu@taichi.graphics  with your offline address. We would deliver a hoodie to you.  : ) Looking forward to hearing from you.

**Pull request merged**